src/platform/user/profile/vet360/components/AddressField/AddressEditModal.jsx

```diff
@@ -17,7 +17,7 @@ export default class AddressEditModal extends React.Component {
       ...this.props.field.value,
       [field]: value,
     };
-    this.props.onChange(newFieldValue, field);
+    this.props.onChange(newFieldValue, field, true);
```

Comment:
This prevents running validation on every keystroke. If we run validation on every change to the input the user will be shown an error while they are in the process of entering their valid zip code. The validation will still run on blur and when they try to submit.

Reply:
This answers the question I was just typing out lol.